modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -562,7 +562,7 @@ private JSONArray _getFiltersJSONArray(
 					).put(
 						"id", properties.get("fieldName")
 					).put(
-						"label", properties.get("name")
+						"label", properties.get("label")
```

Comment:
Not 100% sure, but I think this should be done like in here -> https://github.com/liferay-frontend/liferay-portal/pull/3683/commits/d4dfca99946e029275de9f86ce6dc644d277a014#diff-14ac39b8c7158c0601a7ccd2a2cacb7d24afe76cdd503e7f3c4921a11e3d22c9R518

@markocikos Am I right?  :thinking:

Reply:
Turns out _getValue was added by @juanjofgliferay , so I guess we don't need to ask @markocikos :sweat_smile:

Reply:
@izaera, you're right, one more time. That's the case to get a  default label in the filter